packages/quick_actions/quick_actions_ios/ios/quick_actions_ios/Package.swift

```diff
@@ -0,0 +1,27 @@
+// swift-tools-version: 5.9
+
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import PackageDescription
+
+let package = Package(
+  name: "quick_actions_ios",
+  platforms: [
+    .iOS("12.0"),
```

Comment:
[swift format error](https://ci.chromium.org/ui/p/flutter/builders/try/Mac_x64%20check_podspecs/12657/overview):
```suggestion
    .iOS("12.0")
```